utils/clflags.ml

```diff
@@ -87,6 +87,7 @@ and use_prims = ref ""                  (* -use-prims ... *)
 and use_runtime = ref ""                (* -use-runtime ... *)
 and plugin = ref false                  (* -plugin ... *)
 and principal = ref false               (* -principal *)
+and print_variance = ref false          (* -print-variance *)
```

Comment:
I would propose to move the definition just below `print_types` (and to update the comment).

Reply:
Thank you for spotting. I will fix it.